nilearn/reporting/sphinx_report.py

```diff
@@ -0,0 +1,48 @@
+# Scraper for sphinx-gallery
+# Inspired from https://github.com/mne-tools/mne-python/
+import weakref
+
+from nilearn.reporting import HTMLDocument
+
+
+_SCRAPER_TEXT = '''
+.. only:: builder_html
+
+    .. container:: row sg-report
+
+        .. raw:: html
+
+            {0}
+
+'''  # noqa: E501
+
+
+def indent_and_espace(text, amount=12):
```

Comment:
Did you mean `_espace` or `_escape`?

Reply:
Good catch ! I think `escape` is right, here 👍